WooCommerce/Classes/ViewRelated/Products/Edit Product/DefaultProductFormTableViewModel.swift

```diff
@@ -555,7 +555,7 @@ private extension DefaultProductFormTableViewModel {
                                                     comment: "Format of the sale period on the Price Settings row until a certain date")
 
         // Reviews
-        static let emptyReviews = NSLocalizedString("No ratings",
+        static let emptyReviews = NSLocalizedString("No (approved) reviews",
                                                     comment: "Placeholder for empty product ratings")
```

Comment:
Nit: should we also update the comment for the string too?

Reply:
Thank you for the suggestion! I can see why updating the comment for the string might be helpful, but in this case, I think the comment accurately describes the intended use of the string. The comment is intended for developers/translators, and it helps them understand where and why the string is used. Since the updated string still serves the same purpose as before, I think the existing comment is still relevant and useful.